### PR TITLE
Fix multiple problems with mishandling of missing entries in compute_…

### PR DESCRIPTION
**Pull request by lfrancioli**
…coverage_stats in sparse_mt.py

### CODE REVIEW

File: gnomad/utils/sparse_mt.py [hunk 1]
```diff
@@ -654,7 +654,7 @@ def compute_coverage_stats(
     mt = mt.select_entries("END", "DP")
     col_key_fields = list(mt.col_key)
     t = mt._localize_entries("__entries", "__cols")
-    t = t.join(reference_ht.annotate(_in_ref=True), how="outer")
+    t = t.join(reference_ht.select(_in_ref=True).key_by("locus"), how="outer")
```
[the remark this answers is not in the document]
Comment (lfrancioli) [reply gnomad/utils/sparse_mt.py]: Good catch! fixed.

**Comment by lfrancioli** [comment]
@ch-kr , I know you already approved it but I had to make a few changes for this to run (compute efficiency) and added the new version of coverage to the resources so if you don't mind it'd be great if you could take a quick look at that last commit!

File: gnomad/utils/reference_genome.py [hunk 2]
```diff
@@ -49,11 +49,13 @@ def get_reference_ht(
...
 
-    context = None
+    context = []
     for contig in contigs:
+        n_partitions = max(1, int(ref.contig_length(contig) / 5000000))
+        logger.info(f"Creating reference contig {contig} with {n_partitions} partitions.")
         _context = hl.utils.range_table(
             ref.contig_length(contig),
-            n_partitions=int(ref.contig_length(contig) / 5000000),
+            n_partitions=n_partitions
```
Comment (ch-kr) [inline gnomad/utils/reference_genome.py:58]: was this creating tables with 0 partitions before you added the line of code with `max`?

Comment (lfrancioli) [reply gnomad/utils/reference_genome.py:58]: yes exactly! I got into troubles when I forgot to remove all the silly `GL*` contigs

**Pull request merged**